test/lambdaconnect_model/utils_test.clj

```diff
@@ -0,0 +1,11 @@
+(ns lambdaconnect-model.utils-test
+  (:require [clojure.test :refer [deftest is testing]] 
+            [lambdaconnect-model.core :as mp] 
+            [clojure.pprint :as pprint :refer [pprint]]
+            [lambdaconnect-model.utils :refer [relevant-tags]]))
+
+(deftest relevant-tags-test
+  (let [contraint-to-be-tested '(and [= :tmp-field :tag0] (or [= :tmp-field :tag1] (and (or-join [= :tmp-field  :tag4] [= :tmp-field :tag2]) (not [= :tmp-field :tag3])))) 
+        expected-results (reduce (fn [cur-set new-idx] (conj cur-set (keyword (str "tag" new-idx)))) #{} (range 5))]
```

Comment:
Jak przeczytałem to jak to napisałeś to się zdziwiłem, ja bym to napisał tak:
```
(->> (range 5)
  (map #(keyword (str "tag" %)))
  set)
```
Staram się jak najrzadziej używać `reduce` - wydaje mi się jakiś mało czytelny. Ale to tylko luźna uwaga, może zostać tak jak jest.